content/academy/api_scraping/graphql_scraping.md

```diff
@@ -0,0 +1,29 @@
+---
+title: GraphQL scraping
+description: Dig into the topic of scraping APIs which use the latest and greatest API technology - GraphQL. GraphQL APIs are very different from regular REST APIs.
+menuWeight: 2.2
+paths:
+    - api-scraping/graphql-scraping
+---
+
+# [](#graphql-scraping) GraphQL scraping
+
+[GraphQL](https://graphql.org/) APIs different from the regular [REST](https://www.redhat.com/en/topics/api/what-is-a-rest-api)ful APIs you're likely familiar with, which means that different methods and tooling are used to scrape them. This course will teach you everything you need to know about GraphQL to scrape an API built with it.
+
```

Comment:
I would add a section explaining Advantages & disadvantages of GraphQL scraping compared to regular API scraping. From top of my head:

Advantages:
- Allows trimming down unused properties to speed up the responses and make it easier for the target site
- Allows getting extra data that is not natively available on the target site

Disadvantages:
- More complex to understand
- Introspection is only enabled for some sites which makes it harder to reap the benefits